migration.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Migrate to Pulp 3
+sidebar: home_sidebar
+permalink: /migrate-to-pulp-3/
+toc: false
+---
+
+## Migrate from Pulp 2 to Pulp 3
+
+For many years, the Pulp team have worked to gather and implement feedback from user deployments of Pulp 2. The solution to much of the feedback lay outside of the constraints of the Pulp 2 architecture. This led to the development of Pulp 3, which is not only an update of Pulp 2, but a complete technical overhaul to provide the most effective repository management platform.
+
+At the moment, the Pulp team supports both Pulp 2 and Pulp 3. However, at some point in the future, support will end for Pulp 2 and focus exclusively on Pulp 3.  
```

Comment:
I wonder if we should say that Pulp 2 is in maintenance mode and not mentioning its support. Or is it too early to state that since we haven't decided/announced it anywhere?

@bmbouter , @dkliban , @ipanova , thought?

Reply:
I agree. Let's call out the maintenance mode.

Reply:
+1 to call it a maintenance mode, and yes we have not announced this anywhere officially ( i think) howe on the BZs and issues we do state that. Should we maybe create a blog post?

Reply:
I'll write a post and push the draft on Monday! Thank you!

Reply:
+1 to maintenance mode

+1 to a blog post.

Reply:
+1 to maintenance mode

+1 to a blog post.